src/tests/End2EndTestsMain.cpp

```diff
@@ -16,7 +16,27 @@
 #include "tests/WebnnTest.h"
 
 int main(int argc, char** argv) {
-    InitWebnnEnd2EndTestEnvironment();
+    std::string device = "Default";
+    for (int i = 1; i < argc; ++i) {
+        if (strcmp("-d", argv[i]) == 0 && i + 1 < argc) {
+            device = argv[i + 1];
+        }
+    }
+
+    ml::ContextOptions options;
+    if (device == "CPU") {
```

Comment:
Should we ignore case sensitivity?

Reply:
> Should we ignore case sensitivity?

Thanks! I suggest keeping case sensitive to align with OpenVINO native benchmark. @fujunwei

Reply:
I would suggest to to ignore the case and use the lowercase by default, such as "cpu", "gpu" and "default". That aligns to webnn spec.

Reply:
I have updated to ignore the case , PTAL again, thanks! @huningxin @fujunwei